addon/components/license-picker/component.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
+import _ from 'lodash';
 import layout from './template';
 
+
 export default Ember.Component.extend({
```

Comment:
While you're here, could you add some documentation for this component that includes sample usage? Also it would be good if the component properties were explained, especially the licenses, currentValues and autosave properties.